### PR TITLE
added an iterator for get_pages method

### PR DESCRIPTION
**Pull request by coreGreenberet**
Async/DashboardAPI got a new parameter called "use_iterator_for_get_pages".

If set to true, the api will return an iterator instead of a whole list/dict with all entries.
For backwards compatibility this parameter is set to False by default.

To make the iterator work for getNetworkEvents, I've removed the surrounding informations like pageStartAt, pageEndAt.

The issue without having an iterator got already discussed in the community forum: https://community.meraki.com/t5/Developers-APIs/networks-getNetworkClients-fetch-in-batches/m-p/110536#M4446

the asyncio even got a performance boost that way, as the iterator method will load the next page while yielding all the items from the list.

### CODE REVIEW

File: examples/aio_get_pages_iterator.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 # for example, in Terminal on macOS:  export MERAKI_DASHBOARD_API_KEY=66839003d2861bc302b292eb66d3b247709f2d0d
 api_key = ""
 
-ORGANIZATION_ID = ""
-NETWORK_ID = ""
+ORGANIZATION_ID = "619244948763444192"
```
Comment (TKIPisalegacycipher) [inline examples/aio_get_pages_iterator.py]: Did you mean to add this org ID?

File: examples/aio_get_pages_iterator.py [hunk 2]
```diff
@@ -14,8 +14,8 @@
 # for example, in Terminal on macOS:  export MERAKI_DASHBOARD_API_KEY=66839003d2861bc302b292eb66d3b247709f2d0d
 api_key = ""
 
-ORGANIZATION_ID = ""
-NETWORK_ID = ""
+ORGANIZATION_ID = "619244948763444192"
+NETWORK_ID = "L_645140646620827530"
```
Comment (TKIPisalegacycipher) [inline examples/aio_get_pages_iterator.py]: Did you mean to add this network ID?

**Comment by coreGreenberet** [comment]
Not really, but they should already be removed in the latest commit

File: examples/aio_get_pages_iterator.py (hunk 2 above)
Comment (coreGreenberet) [reply examples/aio_get_pages_iterator.py]: Not really, but they should already be removed in the latest commit.
Could you do a squash merge please? That way they won't stay in the commit history :)

File: examples/aio_get_pages_iterator.py (hunk 1 above)
Comment (coreGreenberet) [reply examples/aio_get_pages_iterator.py]: Not really, but they should already be removed in the latest commit.
Could you do a squash merge please? That way they won't stay in the commit history :)

**Comment by TKIPisalegacycipher** [comment]
Thanks, looking forward to using this!

**Pull request merged**